cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/SvInjectionConversion.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.elements;
+
+import com.powsybl.cgmes.conversion.Context;
+import com.powsybl.cgmes.model.CgmesTerminal;
+import com.powsybl.iidm.network.*;
+import com.powsybl.triplestore.api.PropertyBag;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class SvInjectionConversion extends AbstractIdentifiedObjectConversion {
+
+    public SvInjectionConversion(PropertyBag p, Context context) {
+        super("SvInjection", p, context);
+        String tn = p.getId("TopologicalNode");
+        Terminal associatedTerminal = context.terminalMapping().findFromTopologicalNode(tn);
+        if (associatedTerminal == null) {
+            cgmesTerminal = context.cgmes().terminal(context.terminalMapping().findCgmesTerminalFromTopologicalNode(tn));
+            if (cgmesTerminal == null || context.cgmes().voltageLevel(cgmesTerminal) == null) {
+                context.invalid(id,
+                        String.format("The CGMES terminal and/or the voltage level associated to the topological node %s linked to the SV injection %s is missing",
+                                tn, id));
+                return;
+            }
+            voltageLevel = context.network().getVoltageLevel(context.cgmes().voltageLevel(cgmesTerminal));
+            if (context.nodeBreaker()) {
+                node = context.nodeMapping().iidmNodeForTerminal(cgmesTerminal, voltageLevel);
+            }
+        } else {
+            voltageLevel = associatedTerminal.getVoltageLevel();
+            if (context.nodeBreaker()) {
+                VoltageLevel vl = associatedTerminal.getVoltageLevel();
+                VoltageLevel.NodeBreakerView nb = vl.getNodeBreakerView();
+                node = nb.getNodeCount();
+                nb.setNodeCount(nb.getNodeCount() + 1);
+                vl.getNodeBreakerView().newInternalConnection()
```

Comment:
We are not calling here `iidmNodeForTerminal` of `Context.NodeMapping`; we are bypassing the insertion of the mapping between cgmes terminals and iidm nodes. Maybe in this case this mapping will not be required later, but it could be useful to have it recorded

Reply:
Same as above, I put all of this in a new method in `NodeMapping` to keep consistency between cgmes terminals and nodes. What do you think? (It would be a "fictive" CGMES terminal since the SV injection is not linked directly to a CGMES terminal but to a Topological Node).